src/bridges/erc4626/VaultBridge.sol

```diff
@@ -0,0 +1,157 @@
+// SPDX-License-Identifier: GPL-2.0-only
+// Copyright 2020 Spilsbury Holdings Ltd
+pragma solidity >=0.6.10 <=0.8.10;
+pragma experimental ABIEncoderV2;
+
+import {SafeMath} from '@openzeppelin/contracts/utils/math/SafeMath.sol';
```

Comment:
All our bridges are targeting solidity >=0.8.4 so please set this version even for this bridge and remove this dependenct.